Kickstarter.xcodeproj/xcshareddata/xcschemes/Kickstarter-iOS.xcscheme

```diff
@@ -129,7 +129,7 @@
          <EnvironmentVariable
             key = "OS_ACTIVITY_MODE"
             value = "disable"
-            isEnabled = "YES">
+            isEnabled = "NO">
```

Comment:
Revert?

Reply:
Yeah. I think we should discuss this with @justinswart  and see if it's worth it to disable this. I know this flag makes the logs less overwhelming, but, at the same time, it hides important informations like crash messages and conflicting constraints. I will revert it for now though.